e2e-tests/pages/super_search_page.py

```diff
@@ -101,13 +119,22 @@ def select_match(self, line_id, match):
         self.find_element(*self._highlighted_text_locator).click()
 
     def field(self, line_id):
-        return self.find_element(self._facet_text_locator[0], self._facet_text_locator[1] % line_id).text
+        return self.find_element(
+            self._facet_text_locator[0],
+            self._facet_text_locator[1] % line_id,
+        ).text
```

Comment:
I probably would have done this instead:

```
elem = self.find_element(
    self._facet_text_locator[0],
    self._facet_text_locator[1] % line_id
)
return elem.text
```

Seems better to break it up with an intermediary step. It's fine as is, too.